render/html/default.py

```diff
@@ -11,6 +11,45 @@
 
 import os, logging, codecs
 
+
+class KeyValueWrapper:
+	"""
+		This holds one Key-Value pair for
+		selectOne/selectMulti Bones.
```

Comment:
Please change this to selectBone. 
selectOneBone and selectMultiBone are deprecated.